ewon_flexy_integration/blueprints/datasynchronization.py

```diff
@@ -0,0 +1,147 @@
+"""Scheduler for historic data synchronization"""
+import os
+import logging
+import requests
+from http.client import CONFLICT, HTTPException
+from ewon_flexy_integration.blueprints.datamailbox import DataMailboxHandler
+from ewon_flexy_integration.models.c8y_ewon_flexy_integration import C8YEwonFlexyIntegration
+from c8y_api.model import Measurement, ManagedObject
+
+from flask import Blueprint, jsonify, request
+from c8y_api.app import CumulocityApp
+
+bp = Blueprint('datasynchronization', __name__)
+logger = logging.getLogger('data synchronization')
+
+
+@bp.route('/executalljobs')
+def execute_all_jobs():
+    """Manually execute synchronization of all jobs via this endpoint.
+    """
+    ##TODO store token in tenant options
+    token = "53edRmx8AACZNyWiMSPpk9JMWy7YMTjQrjVIZ3as79Uiw444nA"
+    dsh = DataSynchronizationHandler()
+    total_jobs_executed = dsh.synchronize_historic_data(token)
+    return "Total jobs executed: " + total_jobs_executed
+
+
+@bp.route('/executejob')
+def execute_job():
+    """Manually execute synchronization for specific job via this endpoint.
+    """
+    ##TODO store token in tenant options
+    token = "53edRmx8AACZNyWiMSPpk9JMWy7YMTjQrjVIZ3as79Uiw444nA"
```

Comment:
The token is already stored in the tenant option as this was done by the user in the settings UI.

So I guess it would make sense that the FE will hand over the token to the BE. Please change the endpoint regarding this new requirement.